public/scripts/power-user.js

```diff
@@ -1063,6 +1068,46 @@ function applyAvatarStyle() {
 
 }
 
+function applyChatMode() {
+    if (!power_user.chat_mode === (null || undefined)) {
+        console.debug('applyChatMode: saw no chat mode defined');
+        return;
+    }
+    console.debug(`poweruser.chat_mode ${power_user.chat_mode}`);
+    $('#chat_mode').val(power_user.chat_mode).prop('selected', true);
+
+    switch (power_user.chat_mode) {
```

Comment:
That whole switch kann literally be four lines. That's me being nitpicky tho :D
Something like
```js
console.log('applying chat mode', Object.entries(chat_modes).find(x => x[1] === power_user.chat_mode));
$('body').toggleClass('waifuMode', power_user.chat_mode !== chat_modes.DEFAULT);
$('body').toggleClass('hLetterBox', power_user.chat_mode === chat_modes.VN_V);
$('body').toggleClass('vLetterBox', power_user.chat_mode === chat_modes.VN_H);
```